docs/sources/http_api/data_source.md

```diff
@@ -202,6 +202,7 @@ Content-Type: application/json
 
 {"id":1,"message":"Datasource added", "name": "test_datasource"}
 ```
+> NOTE: `password` and `basicAuthPassword` should be defined under `secureJsonData` in order to be stored in an encrypted blob in the database and no more sent to the browser. Check also the [Encrypting Sensitive Data]({{< relref "../plugins/developing/auth-for-datasources.md/#encrypting-sensitive-data">}}) documentation for more details.
```

Comment:
Does it make sense to include `password` and `basicAuthPassword` in `secureJsonData` as part of your example?